Formula/opencv@2.rb

```diff
@@ -0,0 +1,98 @@
+class OpencvAT2 < Formula
+  desc "Open source computer vision library"
+  homepage "http://opencv.org/"
+  url "https://github.com/opencv/opencv/archive/2.4.13.2.tar.gz"
+  sha256 "4b00c110e6c54943cbbb7cf0d35c5bc148133ab2095ee4aaa0ac0a4f67c58080"
+  revision 1
```

Comment:
this is technically a new formula. we're just going to delete homebrew/science/opencv not migrate it